destinations/data/quickwit.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: internal.odigos.io/v1beta1
+kind: Destination
+metadata:
+  type: quickwit
+  displayName: Quickwit
+  category: self hosted
+spec:
+  image: quickwit.svg
+  signals:
+    traces:
+      supported: true
+    metrics:
+      supported: false
+    logs:
+      supported: true
+  fields:
+    - name: QUICKWIT_URL
+      displayName: Quickwit URL
```

Comment:
Does quickwit supports other receivers as well?
You can consider adding here `OTLP gRPC` so that it is obvious to users which URL is expected. I recently added it to [jaeger](https://github.com/keyval-dev/odigos/blob/415c65d865343e902768bc9fdef95dc1ee2356bd/destinations/data/jaeger.yaml#L18) for more clarity.

Reply:
Done